Sources/NIOPosix/Bootstrap.swift

```diff
@@ -253,6 +255,26 @@ public final class ServerBootstrap {
         return self
     }
 
+    /// Enables multi-path TCP support.
+    ///
+    /// This option is only supported on some systems, and will lead to bind
+    /// failing if the system does not support it. Users are recommended to
+    /// only enable this in response to configuration or feature detection.
+    ///
+    /// - parameters:
+    ///     - value: Whether to enable MPTCP or not.
+    public func enableMPTCP(_ value: Bool) -> Self {
```

Comment:
Since we know these are not available on Darwin OSes does it make sense to mark them as unavailable?